gent_base.theme

```diff
@@ -122,7 +122,7 @@ function gent_base_preprocess_page(&$variables) {
 function gent_base_preprocess_page_hero_image(&$variables) {
   $route = \Drupal::request()->get('_route');
   if ($route !== 'entity.node.canonical') {
-    $variables['attributes']['class'][] = 'overview-page';
+    $variables['attributes']['class'][] = ['overview-page', 'overview-layout'];
```

Comment:
what does this do? 🤔

Reply:
You need this class if you want the .intro class to work...

Reply:
I don't think the `overview-layout` class is intended for this... We should either move the `intro` class to a global level in styleguide; or add some custom styling in the desired project